bittide-instances/tests/Wishbone/Time.hs

```diff
@@ -7,31 +7,29 @@
 
 module Wishbone.Time where
 
+import Clash.Prelude
+
 import Bittide.DoubleBufferedRam
 import Bittide.Instances.Domains
 import Bittide.ProcessingElement
 import Bittide.ProcessingElement.Util
 import Bittide.SharedTypes
 import Bittide.Wishbone
-import Clash.Cores.UART (ValidBaud, uart)
-import Clash.Cores.UART.Extra (MaxBaudRate)
-import Clash.Explicit.Prelude
-import Clash.Explicit.Testbench
-import Clash.Prelude (withClockResetEnable)
-import Clash.Xilinx.ClockGen
 import Control.Monad (forM_)
 import Data.Char
 import Data.Maybe
 import Language.Haskell.TH
 import Project.FilePath
 import Protocols
+import Protocols.Idle
 import System.FilePath
 import Test.Tasty
 import Test.Tasty.HUnit
 import Test.Tasty.TH
 import Text.Parsec
 import Text.Parsec.String
-import VexRiscv
+
+import qualified Protocols.Df as Df
```

Comment:
Copy-pasting my previous comment: 
This module has enough imports to start grouping them IMO. I like the grouping/sorting you did in [DnaPortE2](bittide-instances/tests/Wishbone/DnaPortE2.hs) with:
- Group prelude at the top (including qualified prelude)
- Group all Bittide modules
- Group all other modules
- Group qualified modules at the bottom